pkgs/os-specific/linux/ledger-udev-rules/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ledger-udev-rules";
+  version = "unstable-2019-02-13";
+
+  src = fetchFromGitHub {
+    owner = "LedgerHQ";
+    repo = "udev-rules";
+    rev = "20cc1651eb551c4855aaa56628c77eaeb3031c22";
+    sha256 = "0riydkc4in10pv4qlrvbg3w78qsvxly5caa3zwyqcmsm5fmprqky";
+  };
+
+  configurePhase = false;
+  buildPhase = false;
```

Comment:
@worldofpeace do you know if this is the right way to disable configuring and building?  I see other expressions using `dontBuild` and `dontConfigure`. `dontBuild` is in the manual, but `dontConfigure` isn't.

Reply:
> @worldofpeace do you know if this is the right way to disable configuring and building? I see other expressions using `dontBuild` and `dontConfigure`.

I actually suggested to do that above https://github.com/NixOS/nixpkgs/pull/55741#discussion_r257482429 https://github.com/NixOS/nixpkgs/pull/55741#discussion_r257482424


> `dontBuild` is in the manual, but `dontConfigure` isn't.

Hmm thought it was https://github.com/NixOS/nixpkgs/issues/39958

Reply:
Yeah it actually doesn't have an effect https://github.com/NixOS/nixpkgs/issues/39766#issuecomment-387451287 :laughing: 

nulling the configure phase is fine too. But we should probably fix that.

Reply:
So which one is recommended?

Reply:
I'd use `dontBuild`, and I guess use `dontConfigure` until it actually does something.
I'm pretty sure that the configurePhase won't cause a problem here anyway.